tests/software/bandage/image/test.yml

```diff
@@ -2,7 +2,7 @@
   command: nextflow run ./tests/software/bandage/image -entry test_bandage_image -c tests/config/nextflow.config
   tags:
     - bandage
-    - bandage_image
+    - bandage/image
```

Comment:
So what was the reasoning behind changing this for all modules? Not all modules have been changed because we have at least 110 of those puppies now.

Reply:
Basically anything that a `_` was supposed to represent a `/. That way the `modules lint`can get fed the proper tag. and I had to change them in all the tags because then the filter wouldn't trigger the pytest workflow.

Reply:
Gotcha! 🍬